dashboard-widgets.el

```diff
@@ -20,10 +20,13 @@
 
 ;;; Code:
 
+(require 'cl-lib)
```

Comment:
Why adding this require :question:

Reply:
This is for addressing the compiling warnings, and `cl` is deprecated in 27.

Reply:
I was not implying removing this line. I was just curious why (not using Emacs master version, still in Emacs 26.3). I think it would be great to add this change. Sorry I do not explain myself clear :smile: 

Thank you @seagle0128